.gitignore

```diff
@@ -0,0 +1,55 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
This is a Docker project with no particular sources, I suggest to no pollute the repository with unnecessary files and contents.